docs/sysadmins/ci-cd.md

```diff
@@ -12,18 +12,28 @@ We use CircleCI to manage Continuous Integration and Continuous Deployment.
 
 Each environment has its own set of environment variables which are stored as secrets in the CircleCI Organization Settings under Contexts. 
 
-OPSGENIE_API	
-TM_APP_API_URL
-TM_APP_API_VERSION
-TM_APP_BASE_URL
-TM_CONSUMER_KEY
-TM_CONSUMER_SECRET
-TM_ENABLE_SERVICEWORKER
-TM_MAPBOX_TOKEN
-TM_MATOMO_ENDPOINT
-TM_MATOMO_ID
-TM_ORG_CODE
-TM_ORG_NAME
-TM_ORG_PRIVACY_POLICY_URL
-TM_ORG_URL
-TM_SERVICE_DESK
\ No newline at end of file
+- OPSGENIE_API	
+- TM_APP_API_URL
+- TM_APP_API_VERSION
+- TM_APP_BASE_URL
+- TM_CONSUMER_KEY
+- TM_CONSUMER_SECRET
+- TM_ENABLE_SERVICEWORKER
+- TM_MAPBOX_TOKEN
+- TM_MATOMO_ENDPOINT
+- TM_MATOMO_ID
+- TM_ORG_CODE
+- TM_ORG_NAME
+- TM_ORG_PRIVACY_POLICY_URL
+- TM_ORG_URL
+- TM_SERVICE_DESK
+
+## Automated Tests
+
+For each Pull Request and branch, the CI runs a set of frontend and backend tests. We have a context in place for setting the database called "tasking-manager-testing", with the following environment variables: 
```

Comment:
nit:
Change 
> We have a context in place for setting the database called "tasking-manager-testing",

To:
We have a context in place called "tasking-manager-testing" for setting up the database,